internal/cli/atlas/backup/exports/jobs/decribe.go

```diff
@@ -43,29 +43,29 @@ func (opts *DescribeOpts) initStore(ctx context.Context) func() error {
 	}
 }
 
-var describeTemplate = `ID	EXPORT BUCKET ID	STATE	SNAPSHOT ID
-{{.ID}}	{{.ExportBucketID}}	{{.State}}	{{.SnapshotID}}
+var describeTemplate = `ID	EXPORT JOB ID	STATE	SNAPSHOT ID
+{{.ID}}	{{.ExportJobID}}	{{.State}}	{{.SnapshotID}}
```

Comment:
Can we first audit [cloud_provider_snapshot_export_jobs](https://github.com/mongodb/go-client-mongodb-atlas/blob/bea58b102e62fd4fa8be83c88b89d9f446d259ac/mongodbatlas)/cloud_provider_snapshot_export_jobs.go make sure it matches the [public docs](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Cloud-Backup-Export/operation/returnOneCloudBackupSnapshotExportJob)

For example it seems https://github.com/mongodb/go-client-mongodb-atlas/blob/bea58b102e62fd4fa8be83c88b89d9f446d259ac/mongodbatlas/cloud_provider_snapshot_export_jobs.go#L79 should take job ID 

I worry about this PR as it is because [CloudProviderSnapshotExportJob](https://github.com/mongodb/go-client-mongodb-atlas/blob/bea58b102e62fd4fa8be83c88b89d9f446d259ac/mongodbatlas/cloud_provider_snapshot_export_jobs.go#L158-L170) the struct behind this template doesn't have a `ExportJobID` field and meaning this template would fail to render and flagging this change hasn't been actually tested